src/game/game.erl

```diff
@@ -0,0 +1,44 @@
+-module(game).
+-behaviour(gen_server).
+
+
+%%-define(INTERVAL, 60000). % One minute
+-define(INTERVAL, 1000). % One minute
+
+-export([start_link/1, init/1, handle_call/3, handle_cast/2, handle_info/2, code_change/3, terminate/2]).
+
+-include_lib("stdlib/include/assert.hrl").
+
+
+%% TODO state record
+
+start_link(Verifiers) ->
+  gen_server:start_link(?MODULE, Verifiers, []).
+
+init(Verifiers) ->
+  io:format("starting game with verifiers ~p~n",[Verifiers]),
+  {ok, _TRef} = timer:send_interval(?INTERVAL, timer_trigger),
```

Comment:
Didn't know this was a thing :o 
How do you stop it? o:

Reply:
I think you can do something like timer:stop(_TRef)

Reply:
or kill the process that started it 😱